unified-runtime/source/adapters/level_zero/v2/image.cpp

```diff
@@ -30,4 +30,28 @@ urBindlessImagesImageFreeExp([[maybe_unused]] ur_context_handle_t hContext,
   return UR_RESULT_SUCCESS;
 }
 
+// ur_result_t urBindlessImagesGetImageMemoryHandleTypeSupportExp(
```

Comment:
Why is this commented out? Should this be removed? I see it's already implemented in image_common.

Reply:
Missed during recent merge commit. Removed.